app/models/cms/builtin.rb

```diff
@@ -55,14 +55,8 @@ class CMS::Builtin < CMS::BasePage
   validates :system_name, presence: true
 
   # TODO: this is a quick fix: we should set the liquid enabled attribute to true when creating builtin templates
-  # in rails 4.2 use the attribute api
-  def read_attribute(name)
-    case name
-    when 'liquid_enabled'.freeze
-        true
-    else
-        super
-    end
```

Comment:
shouldn't this stay? This attribute may be read in another way perhaps? No only through `liquid_enabled`?

Reply:
I now see your other comment. But then wh override `liquid_enabled?` instead of `liquid_enabled`?

Reply:
You mean `liquid_enabled` instead of `liquid_enabled?`, right?

Yeah, I thought about that... and I'm not sure... but then I saw there is also this https://github.com/3scale/porta/blob/rails70/app/models/cms/builtin.rb#L145-L148

(it inherits from another superclass, so it needs its own method), and it's with `?`

Let's see if it works.

Reply:
Maybe a callback to set it to true?